index.php

```diff
@@ -8,8 +8,10 @@
 */
 
 require 'multiCurl.php';
+require 'getAndParseWikipediaPOI.php';
+require 'getAndParseWikivoyageGuides.php';
 
-error_reporting(0); // No need error reporting, or else it will crash the JSON export
+error_reporting(E_ALL); // No need error reporting, or else it will crash the JSON export
```

Comment:
We should move this to a config file, to allow easier debugging (and other conf we may add in the future).